### ISSUE TITLE
Enable loading of vanilla loss functions for various frameworks

### ISSUE THREAD

**Issue by nrweir** [CosmiQ/cw-nets#48]
Required frameworks:
- [ ] Darknet (@avanetten is this necessary if we're going to use YOLT?)
- [ ] TensorFlow/Keras
- [ ] PyTorch

**Comment by nrweir** [comment]
Issue moved to [CosmiQ/solaris #31](https://github.com/CosmiQ/solaris/issues/31) via [**ZenHub**](https://www.zenhub.com/)
